Create Mentor Page

Isha988 opened the pull request:
## Related Issue
Closes #168 
<!--
Provide information about the issue or bug that is being addressed.
-->


## Description of Changes
Started building Mentor page
<!--
Clearly and concisely describe the modifications made to successfully resolve the assigned issue. Include any pertinent information about new files or any other relevant details.

For example:
- Added a new function to handle XYZ.
- Updated the ABC module to fix the bug.
- Refactored code in the DEF class for improved performance.
-->

## Checklist:

<!--
Mark the checkboxes to indicate completion. Example:
- [x] My code follows the style guidelines of this project.
-->

- [ ] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| ![Original screenshot](link) | ![Updated screenshot](link) |

Please provide any necessary screenshots to illustrate the changes made.

Isha988 commented:
Hey, @anmode  and @Abhinavcode13 , I have started building mentor page . Its route is '/mentor'.
One request, would you please provide me the graphics and content to add  on the page.

anmode commented:
So @Isha988 folder structure be like..... mentorR.page.js.  actually we have currently `mentor.page.js` it should be `mentorProfile.page.js`. We will change is meanwhile you create a page `mentorR.page.js.`. and create a folder in component as `mentorPage` store all the components of this page in it .
hope you get that.

anmode commented:
For the content shortly I'll update in this pR. Please If anyone collaborate with you on this pR to create graphics. 
Okay give me time I'll update graphics.

Isha988 commented:
> So @Isha988 actually we have currently `mentors.page.js` it should be `mentorsList.page.js`. We will change, you created page `mentor.page.js.` that's cool and please create a folder in component as `mentorPage` store all the components of this page in it .

Okay, I will

Isha988 commented:
@anmode I have added a new section at the last and also updated the companies logo, please have a look at that. 
And if you want any new sections or any updates so please let me know.

anmode commented on components/mentorPage/Company.js:
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function Company({ imgSrc, name }) {
+  return (
+    <div className="row align-items-center">
+      <div className="col text-center text-lg-right p-0 mr-10">
+        <img src={imgSrc} height={30} />
+      </div>
+      <div className="col d-none d-lg-block text-left p-0">
+        <p className="font-weight-bold text-muted m-0">{name}</p>
+      </div>
+    </div>
+  );
+}
+
+export default Company;
```
compony <-----> Intern

anmode replied:
@Isha988 ?

anmode reviewed (approved):
Impressive.
1. FAQ Section needed
2. No worries for the content we will add that.
3. `png` to `svg` or `webp` if possible

Isha988 commented:
> Impressive.
> 
> 1. FAQ Section needed
> 2. No worries for the content we will add that.
> 3. `png` to `svg` or `webp` if possible

Okay, Will create a FAQ section.
I will search for webp or svg logo files, if I get so I will change them.

Abhinavcode13 commented:
> > Impressive.
> > 
> > 1. FAQ Section needed
> > 2. No worries for the content we will add that.
> > 3. `png` to `svg` or `webp` if possible
> 
> Okay, Will create a FAQ section. I will search for webp or svg logo files, if I get so I will change them.

Cool @Isha988 ;)

anmode commented:
A update @Isha988 ... we are using this color `845ec2`. make sure to setup theme button color and on click just make it little dark.

Isha988 commented:
> A update @Isha988 ... we are using this color `#845ec2`. make sure to setup theme button color and on click just make it little dark.

Okay

Isha988 commented:
@anmode @Abhinavcode13 I have added the FAQ section, changed company logo images to svg and changed the color of button. Have a look at them

anmode commented on components/mentorPage/FAQ.js:
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+
+function FAQ({ ques, ans }) {
+  const [open, setOpen] = useState(false);
+  const toggleAns = () => setOpen(!open);
+
+  return (
+    <div className="gray-bg mb-20 rounded container py-5 text-justify">
+      <div className="row justify-content-between align-items-center">
+        <p className="h2 col-10 col-sm-11">{ques}</p>
+        <div className="col-1 cursor-pointer" onClick={toggleAns}>
+          {open ? (
+            <i class="fas fa-chevron-up"  />
+          ) : (
```
Make it `setOpen` not only click on pointer but to on the whole container....clicking anywhere open

anmode reviewed (approved):
Minor Changes and it is good to merge.. Great @Isha988 .

Isha988 commented:
@anmode Required changes made.

Isha988 commented:
`company.js` contains the company component.

anmode reviewed (commented):
great!
Some more minor coments:
1. We want to show `our top mentor did interne at ` instean `our top mentor works at`
2. At one time only a faq should be open. for eg. I open first one and then when i open 2nd fag it should be close. ( I am not going in logic for now)

All things are good!

Isha988 commented:
> great! Some more minor coments:
> 
> 1. We want to show `our top mentor did interne at ` instean `our top mentor works at`
> 2. At one time only a faq should be open. for eg. I open first one and then when i open 2nd fag it should be close. ( I am not going in logic for now)
> 3. Important one - Mobile optimization
> 
> All things are good!

Mobile optimization? Does it means responsive?

Abhinavcode13 commented:
YEAH @Isha988

Isha988 commented:
> YEAH @Isha988

I think it's already responsive.

anmode commented:
> > YEAH @Isha988
> 
> I think it's already responsive.

yeah I know it is reponsive...but not fully optimized. Just check a little bit. Otherwise, I'll note again all optimization and list here. Ok try and address my point 1 and 2 please.
btw really impressed by your work.

Isha988 commented:
> > > YEAH @Isha988
> > 
> > 
> > I think it's already responsive.
> 
> yeah I know it is reponsive...but not fully optimized. Just check a little bit. Otherwise, I'll note again all optimization and list here. Ok try and address my point 1 and 2 please. btw really impressed by your work.

@anmode Made the changes mentioned in points 1 and 2.
For mobile optimization I made few changes have a look at it still if some changes are required so please list them here.

anmode reviewed (approved):
Great, A clean PR. Congrats @Isha988 .
